#goal definition
string temp
---
#result definition
string file_name
string file_name_2ndbest
string face_id
float32 best_match_score
float32 second_best_match_score
---
#feedback definition
float32 time
